resolver/src/test/scala/org/globalnames/resolver/SpecConfig.scala

```diff
@@ -9,10 +9,11 @@ import org.slf4j.{Logger, LoggerFactory}
 
 import scala.sys.process.Process
 
-trait SpecConfig extends WordSpec with Matchers with OptionValues
+trait SpecConfig extends FunSpec with Matchers with OptionValues
                     with BeforeAndAfterEach with BeforeAndAfterAll with ScalaFutures
                     with PatienceConfiguration {
   protected val log: Logger = LoggerFactory.getLogger(getClass)
+  val context = describe _
```

Comment:
please, check out http://doc.scalatest.org/1.7/org/scalatest/FunSpec.html at around words `The recommended style, however, is the functional, pass-all-the-needed-values-in style. Here's an example:`. Could we get more powers to our tests from their approach? Could we substitute `context` with nested `describe` with parentheses in it, or functions as fixtrues?